__tests__/unit/old-structure/tree-fetching/stratetegies/beagle-cache-only.spec.ts

```diff
@@ -91,33 +89,12 @@ describe('Utils: tree fetching (load: beagle-cache-only)', () => {
     expect(nock.isDone()).toBe(true)
   })
 
-
-  it('should throw error if received 304 from bff and doesnt find item in the storage', async () => {
-    const metadata = {
-      beagleHash: 'testing',
-      requestTime: 20203030,
-      ttl: '5'
-    }
-    
-    nock(basePath).get(path).reply(304, undefined,  { 'beagle-hash': 'testing', 'cache-control': 'max-age=5' })
-    const onChangeTree = jest.fn()
-    const loadParams = { url, onChangeTree, strategy, retry }
-    await expect(viewClient.load(loadParams)).rejects.toEqual([
-      new BeagleExpiredCacheError(url),
-      new BeagleCacheError(url)
-    ])
-    expect(storage.getItem).toHaveBeenCalledWith(cacheKey)
-    expect(storage.getItem).toHaveBeenCalledWith(treeKey)
-    expect(nock.isDone()).toBe(true)
-  })
-
   it('should not fallback to cache and throw errors', async () => {
     storage.setItem(treeKey, JSON.stringify(treeA))
     nock(basePath).get(path).reply(500, JSON.stringify({ error: 'unexpected error' }))
     const loadParams = { url, onChangeTree: jest.fn(), strategy, retry }
     await expect(viewClient.load(loadParams)).rejects.toEqual([
-      new BeagleExpiredCacheError(url),
-      new BeagleNetworkError(url, {} as Response)
+      new BeagleNetworkError(url, {} as Response,500,'GET')
```

Comment:
```suggestion
      new BeagleNetworkError(url, {} as Response, 500, 'GET')
```
It would be nice to add the space after the "," to keep a pattern.